frontend/lib/resolution/call-graph.cpp

```diff
@@ -103,10 +103,14 @@ struct CalledFnCollector {
       collectCalls(re);
 
       // Look for uses of module-scope variables in internal/standard modules,
-      // which we will need to convert.
+      // which we will need to convert. Currently call graph traversal does not
+      // enter into modules when module-scope variables are used, so we have
+      // to explicitly do that here.
```

Comment:
"...and we don't visit them (internal modules) elsewhere."

This is implied by the TODO below, but I'd prefer if it were more clear.